brokerapi/brokers/cloudsql/broker.go

```diff
@@ -453,19 +453,20 @@ func (b *CloudSQLBroker) PollInstance(ctx context.Context, instance models.Servi
 	}
 
 	if instance.OperationType == models.ProvisionOperationType {
-		if err := b.refreshServiceInstanceDetails(ctx, &instance); err != nil {
+		instancePtr := &instance
```

Comment:
Referencing a variable should be an O(1) op, so there's not performance benefit from storing the pointer. "&instance" on its own is pretty clear to read whereas using instancePtr requires someone to lookup the variable definition/instantiation to understand the code.